libpod/container_config.go

```diff
@@ -146,6 +146,8 @@ type ContainerRootFSConfig struct {
 	// working directory if it does not exist. Some OCI runtimes do this by
 	// default, but others do not.
 	CreateWorkingDir bool `json:"createWorkingDir,omitempty"`
+	// Secrets lists secrets to mount into the container
+	Secrets []string `json:"secrets,omitempty"`
```

Comment:
What is this - a list of UUID string we can get the secrets via?

Reply:
Yep, either the secret name or secret ID, both work. On a second thought though, I should prolly change it so that it only has secret name.

Reply:
ID seems a lot safer. I do not want people to be able to delete, and subsequently recreate, secrets and have the container use the new one - that seems like a security risk.